backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -866,24 +866,42 @@ public void logObservations(String experimentRunId, List<Observation> observatio
               session, observations, ExperimentRunEntity.class.getSimpleName(), experimentRunId)
           .forEach(
               observation -> {
+                String kvSql =
+                    "INSERT INTO keyvalue"
+                        + " (entity_name, field_type, kv_key, kv_value, value_type, experiment_run_id)"
+                        + " VALUES (:entity_name, :field_type, :kv_key, :kv_value, :value_type, :experiment_run_id)";
+                final long attributeId =
+                    session
+                        .createNativeQuery(kvSql)
+                        .setParameter("entity_name", ExperimentRunEntity.class.getSimpleName())
```

Comment:
missing for observations

Reply:
Do you mean that observations do not populate this field?

Reply:
correct.

Reply:
:+1: